vuepress/docs/next/tutorials/getting-started/azure-install.md

```diff
@@ -32,141 +30,122 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
+```
+http://YOUR-HOST-NAME/app-builder/
 ```
-10. Access your application using the URL for the corresponding ingress, e.g. `http://EXTERNAL-IP.nip.io/entando-de-app/`
 
 See the [Getting Started guide](../../docs/getting-started/README.md#log-in-to-entando) for helpful login instructions and next steps. 
 
 ## Appendix A - Troubleshooting
 
-If you get an error like: `0/5 nodes are available: 5 node(s) had volume node affinity conflict.` or if your deployment hangs in a situation like this from `kubectl get pods -n entando`
-
-```
-NAME                                                 READY   STATUS      RESTARTS   AGE
-my-aks-app-operator-644697776f-sxtq2                 1/1     Running     0          13m
-quickstart-composite-app-deployer-2guz0n42pc         1/1     Running     0          13m
-quickstart-deployer-jj4njqk4bg                       1/1     Running     0          10m
-quickstart-eci-deployer-t0xktqsonk                   0/1     Completed   0          11m
-quickstart-eci-k8s-svc-deployment-78f64c8d89-7c578   1/1     Running     0          11m
-quickstart-kc-deployer-16gzv3clsj                    0/1     Completed   0          13m
-quickstart-kc-server-deployment-7c9bc65744-g52nx     1/1     Running     0          13m
-quickstart-server-deployment-55fcfc6b68-szvkl        0/3     Pending     0          10m
-```
-
-Double check your availability zones. By default an Azure cluster will include nodes from multiple zones, but Azure may not automatically provision their storage.
+If you get an error like: `0/5 nodes are available: 5 node(s) had volume node affinity conflict` or if several deployments fail to start, then you should check your availability zones. By default, an Azure cluster will include nodes from multiple zones, but Azure may not automatically provision their storage.
 
 You can confirm this error in the AKS console as well:
-
-1. In your cluster select `Workloads` in the left nav
-2. Click on the deployment for your server application. This is `quickstart-server-deployment` by default
-3. Click on the deployment name inside that application. There will be one
+1. In your cluster, select `Workloads` in the left nav
+2. Click on the deployment for your server application. This is `quickstart-deployment` by default.
+3. Click on the deployment name inside that application. There will be one.
 4. Click on the tab labeled `Conditions`
-5. If you see an error that says `0/5 nodes are available: 5 node(s) had volume node affinity conflict.` Then you need to reconfigure
-your cluster to have nodes in one zone or work with your Azure operations team to provision storage to match node affinity.
+5. If you see an error that says `0/5 nodes are available: 5 node(s) had volume node affinity conflict.` You need to reconfigure your cluster to have nodes in one zone or work with your Azure operations team to provision storage to match node affinity.
```

Comment:
"if" needs a "then"

period outside of message (is currently inside the backtick)